src/Fhir.Anonymizer.Core/Constants.cs

```diff
@@ -13,9 +13,11 @@ internal static class Constants
         internal const string InstantTypeName = "instant";
         internal const string AgeTypeName = "Age";
         internal const string BundleTypeName = "Bundle";
+        internal const string ReferenceTypeName = "Reference";
```

Comment:
Is it case sensitive?

Reply:
Yes, data type names are case sensitive.